Sage/carbon
# fix(help): navigate to URL when href is set

Fixes FE-2906, Fixes #3009

### Proposed behaviour
Should open links when href is set in props

### Current behaviour
Does not open as preventDefault is set on the click handler, this is no longer needed

### Checklist



### Testing instructions
Click on the help icon should open a new tab, open the codesandbox view in the popout window otherwise the href wont navigate inside the iframe. Known issue when tooltip is rendered outside visible window, this will cause the href to not work. Ticket has been raised to address this.
